src/simplefit/cleaner.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import numpy as np
+
+def cleaner(input_dataframe,lower_case=False):
+    """
+        Load downloaded data, clean the dataset, remove the NA score. 
+        Clean data(remove Nan rows, strip extra white spaces from column names, and data, convert all column names to lower case, etc)
+        Return train_df.info() and train_df.describe()
```

Comment:
Can you remove this from your documentation? As you are not returning train_df.describe() and info now